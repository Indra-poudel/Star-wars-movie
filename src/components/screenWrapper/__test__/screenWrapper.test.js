import React from 'react';
import {Text} from 'react-native';
import renderer from 'react-test-renderer';
import ScreenWrapper from '../screenWrapper';

jest.mock('@react-navigation/elements', () => ({
  useHeaderHeight: () => 10,
}));

test('Renders snapshot as expected', () => {
  const tree = renderer
    .create(
      <ScreenWrapper>
        <Text>Screen Wrapper Componet</Text>
      </ScreenWrapper>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
