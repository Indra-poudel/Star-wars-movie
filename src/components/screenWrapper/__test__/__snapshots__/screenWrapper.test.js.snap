// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders snapshot as expected 1`] = `
<View
  style={
    Array [
      Object {
        "flexGrow": 1,
        "marginTop": 10,
        "paddingHorizontal": 10,
      },
      undefined,
    ]
  }
>
  <Text>
    Screen Wrapper Componet
  </Text>
</View>
`;
